nikola/data/themes/base/templates/index.tmpl

```diff
@@ -11,10 +11,28 @@
         <link rel="prefetch" href="${posts[0].permalink()}" type="text/html">
     % endif
     ${math.math_styles_ifposts(posts)}
+    %if has_other_languages and other_languages:
+        %for language, classification, _ in other_languages:
+            <link rel="alternate" hreflang="${language}" href="${_link(kind, classification, language)}">
+        %endfor
+    %endif
 </%block>
 
 <%block name="content">
-<%block name="content_header"></%block>
+<%block name="content_header">
+    %if has_other_languages and other_languages:
+        <div class="translationslist translations">
+            <h3 class="translationslist-intro">${messages("Also available in:")}</h3>
+        %for language, classification, name in other_languages:
+            <p><a href="${_link(kind, classification, language)}" rel="alternate">${messages("LANGUAGE", language)}
+            %if kind != 'archive':
```

Comment:
I'll do that.

I'll also improve the feed link generation to use the same information; it needs a slightly different list with the current language included.